python/lsst/dax/imgserv/vo/schemas/VOMetadata-v0.1.xsd

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html><head>
+<title>404 Not Found</title>
```

Comment:
This doesn't seem right?  Checking in a 404 message?

Reply:
This XSD is actually missing from IVOA schema repository, but kept here to remind us the status - this issue has been escalated to their working groups.